lib/tandem/tandemSimulator.js

```diff
@@ -47,6 +47,7 @@ exports.make = function(config){
 
   var currBasal = null;
   var currTimestamp = null;
+  var currStatus = null;
```

Comment:
You're not actually using this... but you should. Eventually we will be reading suspends and resumes, and should track `currStatus` in those terms. (`null` for no info, `suspended` or `resumed`, at least to start)